piprot/piprot.py

```diff
@@ -263,7 +263,7 @@ def main(
     session = FuturesSession()
     results = []
 
-    for req, version, ignore in requirements:
+    for req, version, ignore in list(set(requirements)):
```

Comment:
@sesh Actually, how do you feel about simply checking to see if the verbatim flag is active: if so, do nothing -- otherwise cast the results list to a `set` (as above)?

This way, files will be printed _literally verbatim_ (including duplicates, which seems correct given the intention) when desired, otherwise we don't do duplicate work.

I'll push out a quick commit in a moment so you can see what I mean.